src/ng/components/cell/number/body.cell.number.html

```diff
@@ -0,0 +1,4 @@
+<div ng-switch="$cell.column.format !== null">
```

Comment:
In any case default format changes each number with locale pattern
e.g. for en-US locale it will show ```195213``` as ```195,213```
or we need in custom filter for numbers